.github/workflows/js-tests.yml

```diff
@@ -19,18 +26,18 @@ on:
 
 jobs:
   Tests:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [22.x, 18.x, 16.x]
```

Comment:
end of life of 16.x and 18.x is in the past [see](https://endoflife.date/nodejs) i think we can remove them!

Reply:
I only kept them to match master: https://github.com/inveniosoftware/react-searchkit/blob/master/.github/workflows/js-tests.yml#L26
We need to also update react-forms in that case: https://github.com/inveniosoftware/react-invenio-forms/blob/master/.github/workflows/js-tests.yml
and reflect this change on master as well.